assets/model_monitoring/components/generation_safety_quality/generation_safety_quality_signal_monitor/spec.yaml

```diff
@@ -99,13 +99,28 @@ outputs:
     type: uri_folder
     mode: direct
 jobs:
+  input_schema_adaptor:
+    type: spark
+    component: azureml://registries/azureml/components/gsq_input_schema_adaptor/versions/0.0.2
+    inputs:
+      production_dataset:
+        type: mltable
+        path: ${{parent.inputs.production_data}}
+    outputs:
+      adapted_production_data:
+        type: mltable
+    resources:
+      instance_type: ${{parent.inputs.instance_type}}
```

Comment:
As far as I know we don't support changing the `runtime_version` from the user pov so that's why we don't inherit from parent inputs. I don't think any component shares the `runtime_version` they are all static